modules/core/package.json

```diff
@@ -45,8 +45,8 @@
     "gen-docs": "typedoc"
   },
   "dependencies": {
-    "@bitgo/account-lib": "^2.0.0-rc.1",
-    "@bitgo/statics": "^4.3.0",
+    "@bitgo/account-lib": "^2.0.1-rc.1",
+    "@bitgo/statics": "^5.0.0-rc.2",
```

Comment:
Hum? Why didn't this get updated automatically with `yarn lerna publish ...`

Reply:
I don't know... I bumped statics but I don't think it's necessary. Definitely had to bump account-lib